sky/provision/aws/instance.py

```diff
@@ -165,8 +164,7 @@ def cleanup_ports(
         config=config.Config(retries={'max_attempts': BOTO_MAX_RETRIES}))
     # TODO(tian): Add a function to generate SG name for AWS, then replace here
     # and backend_utils::write_cluster_config
-    sg_name = (f'sky-sg-{common_utils.user_and_hostname_hash()}'
-               f'-{common_utils.truncate_and_hash_cluster_name(cluster_name)}')
+    sg_name = f'sky-sg-{cluster_name}'
```

Comment:
ditto

Reply:
I changed this to read directly from teh `provider_config` for the security group name. Does that make sense?